frontend/src/components/MapView/MapView.tsx

```diff
@@ -10,7 +10,7 @@ import { MapLegend } from '../MapLegend/MapLegend'
 import { ProviderPopup, RepresentativePointPopup } from '../MapTooltip/MapTooltip'
 import './MapView.css'
 
-const { MAPBOX_TOKEN } = process.env
+const MAPBOX_TOKEN = process.env.MAPBOX_TOKEN
```

Comment:
Sure! Basically the webpack-define-plugin just interpolates strings into the source used for compilation and transpilation. So previously, since the webpack config file was specifying `process.env.MAPBOX_TOKEN`, the actual value wasn't being interpolated because `process.env` on L13 doesn't match `process.env.MAPBOX_TOKEN`